docs/content/endpoints/download.md

```diff
@@ -1 +1,8 @@
 # /download
+
+To download a song, all you need is the Beatmap ID.
+
+## /download/key/{Beatmap Key}
+_Example: https://beatsaver.com/api/download/key/141_
```

Comment:
> This applies to all added examples

This repo is not associated with `https://beatsaver.com` other than they use this code. Please remove all references to that domain as it's not the single source of truth, other people can run the BeatSaver software too.